packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1187,8 +1188,22 @@ class WidgetsApp extends StatefulWidget {
   /// and view what widgets and render objects associated with it. An outline of
   /// the selected widget and some summary information is shown on device and
   /// more detailed information is shown in the IDE or DevTools.
+  @Deprecated(
+    'Use debugShowWidgetInspectorOverrideNotifier.value instead. '
+    'This feature was deprecated after v3.13.0-19.0.pre.'
+  )
   static bool debugShowWidgetInspectorOverride = false;
 
+  /// If true, the WidgetInspector is added to the tree and made visible.
+  ///
+  /// Used by the `debugShowWidgetInspector` debugging extension.
+  ///
+  /// The inspector allows you to select a location on your device or emulator
+  /// and view what widgets and render objects associated with it. An outline of
+  /// the selected widget and some summary information is shown on device and
+  /// more detailed information is shown in the IDE or DevTools.
+  static ValueNotifier<bool> debugShowWidgetInspectorOverrideNotifier = ValueNotifier<bool>(false);
```

Comment:
Fyi @goderbauer. I'd like your opinion on the naming convention to use for a debug override like this.
If it wasn't for the conflict with the deprecated name, would we want to call this just
debugShowWeidgetInspectorOverride even though the type is ValueNotifier?

Reply:
Why the "override" in this name? What is getting overridden? The answer should be added to the docs.

Wondering: Why does the Notifier need to be part of the public API? It seems like it could be private, only exposing a public getter/setter to change its value since there do not appear to be any clients outside of this file that need to listen to the listenable? The public getter/setter could be the existing `debugShowWidgetInspectorOverride`, so there wouldn't be a need for new API surface at all...